### PR TITLE
HW1 submission (Takuya Matsumoto)

### PR DESCRIPTION
**Pull request by tkymtsmt**
I edited the README to add my Bio such as my name, my favorite subjects, etc.

- What is easy
    - Markdown syntax because I often take a note with R markdown and Jupyter notebook. 

- What is difficult
   - I have used R studio and Github (desktop), but I've never use Github on R studio. Also, I've never used the R studio server. At first, it was a little confusing, but now I find it so helpful.

- Useful material
    - I read only tutorials uploaded on the class website. That may be the most useful instruction.

### CODE REVIEW

**Comment by oneforae** [comment]
I think you did a good job! There's a good variety of information. It looks clean, simple, and easy to read. Good job on the embedded link, as well.

**Comment by jerrashi** [comment]
I liked how thorough you were throughout the process of updating the readme file. I also think your pull request is very thorough. Lastly, I liked reading your bio! Very concise and descriptive.

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+I'm excited to learn R through this course! Here's my Bio ,etc.
+
+### (1) Bio
+
+- My name is **Takuya Matsumoto**, a 2nd year *MPP* student at [Harris School of Public Policy](https://harris.uchicago.edu/).
```
Comment (nearridge) [inline README.md:7]: Nice use of emphasis and a link in one sentence!

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+- From Tokyo in Japan
+
+### (2) My favorite subject
+
+- My favorite subjects are
+    - **Econometrics**
```
Comment (nearridge) [inline README.md:13]: cool!

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |   x        |              |            |
| **Presentation**            |           |         x     |            |
| **Achievement, creativity** |           |     x         |            |
| **Reproducibility**         |     x      |              |            |

Remarks: Great job including extensive and specific commit messages in Git. It makes it easy to see what you did when.

**Pull request closed without merging**